apps/frontend/src/utils/constants.ts

```diff
@@ -17,6 +17,8 @@ export const sovrynAlphaLinks = {
   [Environments.Mainnet]: 'https://alpha.sovryn.app/',
   [Environments.Testnet]: 'https://alpha-test.sovryn.app',
 };
+export const sovrynStagingLink = 'https://staging.sovryn.app';
+export const sovrynAlphaStagingLink = 'https://alpha-staging.sovryn.app';
 
 export const sovrynWikiLinks = {
```

Comment:
```suggestion
  [Environments.Testnet]: 'https://alpha-test.sovryn.app',
};
export const sovrynAlphaStagingLink = 'https://alpha-staging.sovryn.app';

export const sovrynStagingLink = 'https://staging.sovryn.app';

```

Sorry I wasn't clear, like this - just the alpha link :)